lib/smart_proxy_remote_execution_ssh/runners/polling_script_runner.rb

```diff
@@ -132,7 +132,7 @@ def cleanup
     def destroy_session
       if @session
         @logger.debug("Closing session with #{@ssh_user}@#{@host}")
-        @session.close
+        system("/usr/bin/ssh", @host, "-o", "User=#{@ssh_user}", "-o", "ControlPath=#{@local_working_dir}/ssh-%r@%h", "-O", "exit")
```

Comment:
When using `system`, all output the command produces is printed to output streams of the parent process, it should either be silenced or forwarded to proper logs.